examples/workers/l4/compute_then_tload_mixed_l3/main.py

```
@@ -245,45 +245,54 @@ def _parse_args() -> argparse.Namespace:
 
-def main() -> int:  # noqa: PLR0915 -- one linear two-phase scenario; splitting it would scatter the phase ordering this example exists to show
+def run(  # noqa: PLR0915 -- one linear two-phase scenario; splitting it would scatter the phase ordering this example exists to show
+    *,
+    remote: str,
+    local_devices: str,
+    remote_devices: str,
+    platform: str = "a2a3",
+    runtime: str = "tensormap_and_ringbuffer",
+    comm_profile: str = "a3-fabric-v1",
+    session_timeout: float = 120.0,
+    session_listen_host: str = "0.0.0.0",
```

Ruff (0.16.1)

[error] Possible binding to all interfaces

(S104)